hcconfig/interface.py

```diff
@@ -1,12 +1,16 @@
 from collections import namedtuple
 from enum import Enum
 import os
+import sys
 from pathlib import Path
 import re
 
 import cmd
 import click
+import serial
 from serial import Serial
+from serial.tools.list_ports import comports
```

Comment:
This import is no longer required, right?

Reply:
(again, not your fault... :joy: )

Reply:
Done